### DISCUSSION TITLE
Error while syncing

### DISCUSSION THREAD

**Discussion by La1x** [remotely-save/remotely-save#166] [Q&A]
Hi team!

I am trying to use plugin for sync my windows pc with ios.
I configured webdav for Yandex.Disk. First syncing was successful. 
But second try (without any changes in notes) ended with error.

Can you help me to figure out what was wrong?

Some output from console:
```
remotely-save-1660119543293: abort sync, triggerSource=manual, error while syncing
<...>
AggregateError: 
    Error: 3 - Knowledge notes/Software_architecture_&_desing/: Request failed, status 405
        at eval (plugin:remotely-save:211:314339)
<...>
    Error: too many errors, stop the remaining tasks
        at eval (plugin:remotely-save:211:314443)
        at Generator.next (<anonymous>)
        at a (plugin:remotely-save:177:280327)
    at eval (plugin:remotely-save:211:314499)
    at Generator.next (<anonymous>)
    at a (plugin:remotely-save:177:280327)
```

**Comment by Evgeny1503** [comment]
I have such a situation. Still no solution found? But I have 403 error.

**Reply by Evgeny1503** [reply]
I found a solution. The permissions for user on the webdav server were set incorrectly.

**Reply by La1x** [reply]
Hi!
I didnt find a solution. I think this cause by yandex.disk's restrictions to webdav protocol.
Which permissions were set incirrectly in your case? Do you use yandex.disk?

**Reply by Evgeny1503** [reply]
Нет, webdav у меня настроен на домашнем роутере. Права доступа были неверно настроены для пользователя Obsidian(указывается в настройках remotely save). Неправильно было именно на стороне сервера, у пользователя не было прав на запись/чтение для папки синхронизации. Вероятно в вашем случае нужно через настройки Яндекса дать разрешение на запись/чтение к необходимому каталогу для юзера(не знаю как у Яндекса это делается).

**Reply by Evgeny1503** [reply]
В качестве эксперимента проверил у себя на Яндексе. Все работает. Никаких прав доступа там даже настраивать не нужно.

**Comment by JhonDoeMax** [comment]
эту проблеми удалось решить, убрав вложенность папок.
У плагина проблема с рекурсивным проходом по папкам.
Проблема с [Yandex.disk](https://disk.yandex) осталась.
На [Yandex.disk](https://disk.yandex) загружается успешно, но при этом не скачивается.
С [DropBox](https://www.dropbox.com/ru/) работает отлично.

**Reply by alimuradov** [reply]
> эту проблеми удалось решить, убрав вложенность папок. У плагина проблема с рекурсивным проходом по папкам. Проблема с [Yandex.disk](https://disk.yandex) осталась. На [Yandex.disk](https://disk.yandex) загружается успешно, но при этом не скачивается. С [DropBox](https://www.dropbox.com/ru/) работает отлично.

Помог, твой совет, спасибо.
